cosmpy/aerial/client/__init__.py

```diff
@@ -172,6 +184,41 @@ def total_unbonding(self) -> int:
         return sum(map(lambda p: p.amount, self.unbonding_positions))
 
 
+@dataclass
+class Block:
+    """Block."""
+
+    height: int
+    time: datetime
+    chain_id: str
+    tx_hashes: List[str]
+
+    @staticmethod
+    def from_proto(block: Any) -> "Block":
+        """Parse the block.
+
+        :param block: block as Any
+        :return: parsed block as Block
+        """
+        return Block(
+            height=int(block.header.height),
+            time=Block._parse_timestamp(block.header.time),
+            tx_hashes=[sha256(tx).hex().upper() for tx in block.data.txs],
+            chain_id=block.header.chain_id,
+        )
+
+    @staticmethod
+    def _parse_timestamp(timestamp: Timestamp):
+        """Parse the timestamp.
+
+        :param timestamp: timestamp
+        :return: parsed timestamp
+        """
+        return datetime.fromtimestamp(timestamp.seconds, tz=timezone.utc) + timedelta(
+            microseconds=timestamp.nanos // 1000
+        )
```

Comment:
It kind of feels that this method would deserve to me relocated to a place with more generic purpose than `Block` class - normally it ould be somethiong like `utils`, `common` or similar, but it looks like that cosmpy does not have such explicitly named  components and rather uses `__init__.py` files to do such generic purpose things.

Reply:
I had the same feeling, but we should move it together with other stuff in some refactoring PR.